hbase-endpoint/src/test/java/org/apache/hadoop/hbase/coprocessor/TestRegionServerCoprocessorEndpoint.java

```diff
@@ -20,22 +20,14 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-import com.google.protobuf.RpcCallback;
-import com.google.protobuf.RpcController;
-import com.google.protobuf.Service;
```

Comment:
Yeah, so client now has to have hbase-shaded-protobuf on its side and if we change internal pb version we could break the endpoint?

Realistically, the pb3 should probably stay on-the-wire compat so should be concern till pb4 or until we decided we want our pbs to be schema v3 compat.

Reply:
In previous discussion the idea is to keep the protobuf version 'stable' for a whole major release.